components/drafts/panel/panel.scss

```diff
@@ -0,0 +1,18 @@
+.Panel {
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+    background-color: var(--center-channel-bg);
+    border-radius: 4px;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.08);
```

Comment:
We have `--elevation-1` variables let use that

Reply:
Done